specs/Standards/MultiToken/Events.md

```diff
@@ -0,0 +1,271 @@
+# Multi Token Event([NEP-246](https://github.com/near/NEPs/discussions/246))
+
+
+Version `1.0.0`
+
+## Summary
+
+Standard interfaces for Multi Token Contract actions.
+
+## Motivation
+
+MT-driven apps such as marketplaces and videogames perform a few 
+core actions `minting`, `burning`, `transferring`, and 
+`approving tokens for transfer`.
```

Comment:
`approve` action does not send the tokens, so I'm not sure we want to show it as a separate event log type.

Reply:
@MaximusHaximus @frol what do you think?

Reply:
@telezhnaya I just made the executive decision to be in alignment with the other standards. I don't have to have that in for the initial setup.

Reply:
so yanked it